Formula/cmake.rb

```diff
@@ -66,8 +65,19 @@ def install
     (pkgshare/"Modules/Internal/CPack/CPack.OSXScriptLauncher.in").unlink
   end
 
+  def caveats
+    <<~EOS
+      To install the CMake documentation, run:
+        brew install cmake-docs
+    EOS
+  end
+
   test do
     (testpath/"CMakeLists.txt").write("find_package(Ruby)")
     system bin/"cmake", "."
+
+    # These should be supplied in a separate cmake-docs formula.
+    refute_path_exists doc/"html"
+    refute_path_exists man
```

Comment:
@Bo98 Note that this will result in the `system bin/"cmake"` never failing because there's no assertion. `test do` was previously checking to make sure `false` wasn't returned and, after this change, it never will be (from `system`).

Reply:
Fix in https://github.com/Homebrew/homebrew-core/pull/89909

Reply:
I'm wrong, no fix needed: https://github.com/Homebrew/homebrew-core/pull/89909#issuecomment-977896814